docs/UserGuide.md

```diff
@@ -426,24 +477,42 @@ starts on 30 Jan 2022 and ends on 23 Nov 2022 to the 1st student in the displaye
 
 Edits the specified lesson of the specified student in TAB with the indicated changes for specified fields.
 
-Format: `ledit INDEX LESSON_INDEX [recurring/[END_DATE]] [date/START_DATE] [time/TIMERANGE] [subject/SUBJECT] [rates/LESSON_RATES] [f/OUTSTANDING_FEES] [hw/HOMEWORK]… [cancel/CANCEL_DATE]… [uncancel/UNCANCEL_DATE]…​`
+Additional fields to the ones in [Managing lessons](#managing-lessons) are listed below.
+
+Field | Prefix | Constraints | Examples |
+------------------|-------|-----------| --------|
+Cancelled date |`cancel/`| Same constraints as other date fields.<br/> The date to be cancelled must be a valid lesson date.<br>e.g. If the start date of a recurring lesson is `1 Oct 2021`, you can cancel `8 Oct 2021` but not `2 Oct 2021`.|`cancel/20 jan 2022`|
+Uncancelled date |`uncancel/` | Same constraints as other date fields.<br/>The date to be uncancelled must be a date that has already been cancelled.| `uncancel/20 jan 2022`|
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+
+* If you change the start date of the lesson, the cancelled dates that become invalid will be removed.
+
+* Editing the homework set of a lesson will clear all existing pieces of homework and add the newly specified pieces of homework.
+
+</div>
+
+Format: `ledit INDEX LESSON_INDEX [recurring/END_DATE] [date/START_DATE] [time/TIMERANGE] [subject/SUBJECT] [rates/LESSON_RATES] [f/OUTSTANDING_FEES] [hw/HOMEWORK]… [cancel/CANCEL_DATE]… [uncancel/UNCANCEL_DATE]…​`
 
 * Edits the lesson of specified `LESSON_INDEX` for the student at the specified `INDEX`.
 
 * The index must be a valid index number shown in the displayed student list.
 
 * The lesson index must be a valid index number shown in the lesson list of the student.
 
-* You can edit all fields of a lesson.
+* When editing homework, all existing pieces of homework of the lesson will be removed and replaced with the pieces of homework specified.<br>
+  e.g. `ledit 2 1 hw/As2` will erase the lesson's original pieces of homework and replace it with the new homework `As2`.
 
-* You cannot change the lesson's type (i.e. recurring and makeup).
+* You can delete all homework of a lesson by typing `hw/` without any arguments. 
 
-* The date to cancel must be a valid lesson date.<br>
-  e.g. If the start date of a recurring lesson is `1 Oct 2021`, you can cancel `8 Oct 2021` but not `2 Oct 2021`.
-  
-* The date to uncancel must be an already cancelled date.
+<div markdown="span" class="alert alert-primary">**:information_source: Note about homework:**<br>
```

Comment:
what is this note for? how is it different from the point above?

Reply:
Highlights the difference in behaviour when u use `hw/`  without any other `hw/` non-empty prefixes and when you use it with them

Reply:
then should the example be multiple homework? I'm a bit confused..

Reply:
Oops, I will remove the example.